gnomad_qc/v5/data_ingestion/federated_validity_checks.py

```diff
@@ -1030,14 +1079,29 @@ def main(args):
         ht = ht.annotate(info=ht.info.annotate(**annotations))
 
         info_dict = {}
+
+        # Add region_flag fields if present.
+        missing_region_flags = []
         if "region_flags" in ht.row:
-            # Add region_flag to info dict.
             for field in REGION_FLAG_FIELDS:
```

Comment:
I just realized we removed nonpar from this constant and didn't add non_par back, so this won't check for that field's presence/absence. should we add it back here?

Reply:
ah I must have missed the non_par, sounds good!